packages/docs/docs/launch-configuration.md

```diff
@@ -129,15 +129,23 @@ Below is an example of how the `launch.json` file could look like with android v
 Instead of letting Radon IDE build your app, you can use scripts (`buildScript` option) or [Expo
 Application Services (EAS)](https://expo.dev/eas) (`eas` option) to do it.
 
-The requirement for scripts is to output the absolute path to the built app as the
-last line of the standard output.
-
-Both `buildScript` and `eas` are objects having `ios` and `android` optional
+The requirement for scripts is to output the absolute path to the built app as
+the last line of the standard output. If custom fingerprint script is used, it
+should output fingerprint as the last line of the standard output. When
+fingerprint changes between invocations, RN IDE will rebuild the project. The
+IDE runs fingerprint quite frequently (i.e., on every file save), so this
+process should be fast and avoid over the network communication.
```

Comment:
We should explain the concept of fingerprint here. Specifically that it is a string that allows us to tell whether new native build needs to be run